extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -595,7 +675,42 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
               visiblePercentageMin: 50,
               continuousTimeMin: 1,
             });
-    return superReturnValue;
+    return frameLoadPromise;
+  }
+
+  /** @override  */
+  unlayoutCallback() {
+    super.unlayoutCallback();
+    this.maybeRemoveListenerForFluid();
+  }
+
+  /**
+   * Postmessages an initial message to the fluid creative.
+   * @visibleForTesting
+   */
+  connectFluidMessagingChannel() {
+    dev().assert(this.iframe.contentWindow,
+        'Frame contentWindow unavailable.');
+    this.iframe.contentWindow./*OK*/postMessage(
+        JSON.stringify(dict({'message': 'connect', 'c': 'sfchannel1'})),
+        SAFEFRAME_ORIGIN);
+  }
+
+  /**
+   * Fires a delayed impression and notifies the Fluid creative that its
+   * container has been resized.
+   * @private
+   */
+  onFluidResize_() {
+    if (this.fluidImpressionUrl_) {
+      this.fireDelayedImpressions(this.fluidImpressionUrl_, false);
```

Comment:
You don't actually need to include false as its the default

Reply:
Done.